cytominer_database/utils.py

```diff
@@ -23,16 +23,7 @@ def find_directories(directory):
     :return: list of subdirectories of ``directory``
 
     """
-    directories = set()
-
-    filenames = glob.glob(os.path.join(directory, '*/'))
-
-    for filename in filenames:
-        pathname = os.path.relpath(filename)
```

Comment:
@shntnu I'm curious if there's a reason behind using `relpath`, here. I couldn't figure it out from how this function is used throughout the codebase. Do you have any insight?

Reply:
gosh - no clue :(

I thought this would break with this PR but it didn't:

    cp -r tests/data_a /tmp/
    cytominer-database ingest -c tests/data_a/config.ini --munge /tmp/data_a/ sqlite:////tmp/data_a.sqlite
    sqlite3 /tmp/data_a.sqlite 'select count(*) from Cells;'
    > 40

So I'm not sure why we had that.

Ok to proceed with this change @mcquin